core-primitives/sgx-runtime-primitives/src/types.rs

```diff
@@ -0,0 +1,39 @@
+use sp_runtime::{
+	generic,
+	traits::{BlakeTwo256, IdentifyAccount, Verify},
+	MultiSignature,
+};
+
+/// The address format for describing accounts.
+pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
+/// Block header type as expected by this sgx-runtime.
+pub type Header = generic::Header<BlockNumber, BlakeTwo256>;
+
+/// An index to a block.
+pub type BlockNumber = u32;
```

Comment:
Yeyy, that's cool. But now the fun part: Consolidate that with all the other definitions. Just like here: 
https://github.com/integritee-network/worker/blob/master/core-primitives/types/src/lib.rs#L45-L61

Reply:
I re-exported from `sgx-runtime-primitives`.
As we've discussed it is worth checking other type duplicates in other places as well. Should it be done in this PR, or in a separate one?

Reply:
I believe for types that are now defined in sgx-runtime-primitives that should be done in this PR.